docs/_blog/bdk_cli_basics_00.md

```diff
@@ -0,0 +1,294 @@
+}
+
+```
+ 
+
+
+***
+### 6a: export PSBT to environment-variable
+ :arrow_forward: `export PSBT="PASTE_PSBT_HERE"`
+
+![](https://i.imgur.com/CEDKcPZ.gif)
+
+
+ 
+
+***
+## Step 7: Sign Transaction (PSBT) 
+
+:arrow_forward: ` bdk-cli wallet --wallet wallet_name --descriptor $my_descriptor sign --psbt $PSBT`
+- DON'T FORGET to COPY the PSBT for the next step
+
+![](https://i.imgur.com/f4o4Ce8.gif)
+
+
+:+1: The output below confirms the command was successful.
+
+
+
+```
+{
+  "is_finalized": true,
+  "psbt": "cHNidP8BAFUBAAAAAak8uMR3UGkAGUKWsq8Mv45qg2fdD93JQRIsa2P0wFloAQAAAAD/////AQfgAQAAAAAAGXapFDRKD0jKFQ7CuQOBdmC5tosTpnAmiKwAAAAAAAEA3gIAAAAAAQFY9sVfEEbyjrHXSlxXDxL+71WOMnsPpVElwk+3E/J9vAAAAAAA/v///wIYZRIAAAAAABYAFBKYf7yF+ss6EFdw2rDZTfdLhep8eOABAAAAAAAWABQd6wBRnX8mBZiJYfvy/FGE+xKc+AJHMEQCIFSIkvEUI9yUgEw4JocRs1aiVsBlKKXrOQaQb3XFqR21AiBqiEVzCVVSRGjckyPDgAQBnOdSzBYR6Rw6KFcCP+E27wEhAwIlXdfM2WYnYa36Hp4MS6YkplBAgBsb1tYG9NiWFWTKzPYhAAEBH3jgAQAAAAAAFgAUHesAUZ1/JgWYiWH78vxRhPsSnPgiAgP80FpaWYQzGzCnNI9blXbei61YpAmtoezMRxpVvBJ6S0gwRQIhALWkBRSJzxuf0od4tPu3qFmEfJ2Y+/QBGtfjSFObWsPeAiA4QJx8Rk5pacrjHv5EOdw6RNHXcdtepFs+m0/Za/h0UQEiBgP80FpaWYQzGzCnNI9blXbei61YpAmtoezMRxpVvBJ6SxgTizKsVAAAgAEAAIAAAACAAAAAAAAAAAABBwABCGwCSDBFAiEAtaQFFInPG5/Sh3i0+7eoWYR8nZj79AEa1+NIU5taw94CIDhAnHxGTmlpyuMe/kQ53DpE0ddx216kWz6bT9lr+HRRASED/NBaWlmEMxswpzSPW5V23outWKQJraHszEcaVbwSeksAAA=="
+}
+
+```
+
+***
+### 7a: export signed psbt to environment variable
+
+:arrow_forward: `export SIGNED_PSBT="Paste_PSBT_HERE"`
+
+![](https://i.imgur.com/VJsl8zR.gif)
+
+
+
+
+
+***
+## Step 8: Broadcast Transaction 
+
+:arrow_forward:  `bdk-cli wallet --wallet wallet_name --descriptor $my_descriptor broadcast --psbt $SIGNED_ PSBT`
+
+
+![](https://i.imgur.com/yQZZk0d.gif)
+
+ 
+:+1: The output below confirms the command was successful.
+
+ ```
+{
+  "txid": "a0877b7ce91ea6d141ba63277673f5bdf0edfdd45f91a39ba1a1ace15f839b52"
+}
+ 
+
+ ```
+
+
+
+ - Verify transcation in the memory pool on testnet [Mempool-testnet!](https://mempool.space/testnet)
+ 
+ ***
+## Resources
+
+### Other Tutorials / Blog
+
+- [ My Summer of Bitcoin Experience](https://blog.summerofbitcoin.org/sob-22-experience-bdk/)
+### Useful links
+- [bip-32:  Hierarchical Deterministic Wallets](https://github.com/bitcoin/bips/blob/master/bip-0032.mediawiki)
+- [bip-39: Mnemonic code for generating deterministic keys](https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki)
+- [bip-44: Multi-Account Hierarchy for Deterministic Wallets](https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki)
+- [bip-84: Derivation scheme for P2WPKH based accounts](https://github.com/bitcoin/bips/blob/master/bip-0084.mediawiki)
+- [What are Descriptors and miniscript?](https://blog.summerofbitcoin.org/miniscript-policy-descriptors-hidden-powers-of-bitcoin/)
+- [Master Private  Key and Extended Private Key](https://bitcoin.stackexchange.com/questions/97242/bip39-tool-bip32-extended-private-key-vs-bip32-root-key)
```

Comment:
One more link for PSBT would be nice..

Reply:
Added it to the list. 
Than you  for all the recommendations, the tutorial looks much better now.